src/KintoID.sol

```diff
@@ -83,12 +89,6 @@ contract KintoID is
         domainSeparator = _domainSeparator();
     }
 
-    function initializeV6() external {
```

Comment:
i thought you had already removed this... or was it on a different contract? 🤔

Reply:
different contract